change/office-ui-fabric-react-2020-02-24-17-30-43-master.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
```

Comment:
Is this a minor? I can't see any API changes or behavior changes.

Reply:
Good point, there were some, but I ended up not needing them anymore. I'll change this to be a minor.